plugins/text-freetype2/text-freetype2.c

```diff
@@ -169,7 +172,7 @@ static void ft2_source_destroy(void *data)
 		FT_Done_Face(srcdata->font_face);
 		srcdata->font_face = NULL;
 	}
-	
+
```

Comment:
Next time, try to create a separate commit if you would like to fix some whitespace issues.  Make sure to check your diff before submitting for any changes that don't seem to apply to your actual commit.